### PR TITLE
feat: link to home page

### PR DESCRIPTION
**Pull request by mortennordseth**
As per the Product Owner's request, I have included a new link enabling users to navigate back to the home page of each organization. This adjustment aligns with the Figma sketches. Additionally, I have modified the page header to span the full width, mirroring the design in the Figma sketches.

Edit: I've updated the back link title to include the name of the home page (e.g. frammr.no, atb.no). 

<details>
<summary>Screenshots</summary>

**Figma**

image

**New website**

image

image

</details>

### CODE REVIEW

File: src/translations/pages/assistant.ts [hunk 1]
```diff
@@ -2,6 +2,11 @@ import { translation as _ } from '@atb/translations/commons';
 
 export const Assistant = {
   title: _('Planlegg reisen', 'Plan travel', 'Planlegg reisa'),
+  homeLink: _(
+    'Tilbake til forsiden',
+    'Back to the front page',
+    'Tilbake til framsida',
```
Comment (mortennordseth) [inline src/translations/pages/assistant.ts:8]: Would it be better to have `Tilbake til frammr.no` here?

Comment (jonasbrunvoll) [reply src/translations/pages/assistant.ts:8]: I belive "Tilbake til frammr.no" would be a slightly clearer explanation.

**Review by jonasbrunvoll — APPROVED** [review]
Looks great🚀

File: src/page-modules/assistant/layout.tsx [hunk 2]
```diff
@@ -77,8 +78,18 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
...
+
   return (
     <div>
+      <div className={style.homeLink__container}>
+        <ButtonLink
+          mode="transparent"
+          href={urls.homePageUrl.href}
+          title={t(PageText.Assistant.homeLink(urls.homePageUrl.name))}
+          icon={{ left: <MonoIcon icon="navigation/ArrowLeft" /> }}
+        />{' '}
```
Comment (adriansberg) [inline src/page-modules/assistant/layout.tsx:91]:
```suggestion
        />
```

**Review by adriansberg — APPROVED** [review]

**Pull request merged**